src/components/Navigation.stories.js

```diff
@@ -0,0 +1,10 @@
+import Navigation from "./Navigation";
+
+export default {
+  title: "components/Navigation",
+  component: Navigation,
+};
+const Template = (args) => <Navigation {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {};
```

Comment:
Sry i dont know how to comment the img, i think you uploaded this picture 2 times src/img/joel2.jpg and src/img/joel.jpg but still the Same picture

Reply:
Thanks Zaid!! :)

Reply:
😃 you welcome